network/rpc/common/src/hash.rs

```diff
@@ -11,7 +11,7 @@ use sha3::{
 /// Prefixes the program bytes when computing hash.
 pub const PKG_VERSION: &str = env!("CARGO_PKG_VERSION");
 
-type HashBytes = [u8; <Sha3_256Core as OutputSizeUser>::OutputSize::INT];
+type HashBytes = [u8; <Sha3_256Core as OutputSizeUser>::OutputSize::USIZE];
```

Comment:
This is the same associated constant, but specified directly for the type instead of using the trait. The size of the array is checked to be of `usize` type, therefore I don't follow why the usage is "incorrect"